pkmn.py

```diff
@@ -293,10 +293,9 @@ def run_reinforcement_learning():
             observation = env.reset() # reset env
             report_observation = bookkeeper.construct_observation_handler()
             bookkeeper.signal_episode_completion()
-                  
         if reward != 0: # Pong has either +1 or -1 reward exactly when game ends.
             bookkeeper.signal_game_end(reward)
-
+        print(crash)
```

Comment:
Why is this here?

Reply:
I wanted to just see the first move (in this case, almost always thunderbolt). This line's days are indeed numbered.